Etabs_Adapter/_Engine/Convert/ToCSI.cs

```diff
@@ -102,18 +102,22 @@ public static eMatType ToCSI(this MaterialType materialType)
                     return eMatType.Steel;
                 case MaterialType.Concrete:
                     return eMatType.Concrete;
-                case MaterialType.Timber://no material of this type in ETABS !!! 
-                    return eMatType.Steel;
+                case MaterialType.Timber:
+                    Engine.Reflection.Compute.RecordWarning("ETABS does not contain a definition for Timber materials, it has been set as an equivilant orthotropic material instead");
```

Comment:
Final tweak on these warnings:
"ETABS does not contain a definition for Timber materials,the material has been set to type 'Other'" to match what is shown in the UI:

![image](https://user-images.githubusercontent.com/22005920/74036627-ea314c00-49bc-11ea-8fb1-7bd4711b5dd8.png)